core/arch/aarch64/instr.c

```diff
@@ -335,6 +335,12 @@ reg_is_gpr(reg_id_t reg)
     return (DR_REG_X0 <= reg && reg <= DR_REG_WSP);
 }
 
+bool
+reg_is_simd(reg_id_t reg)
```

Comment:
unused?

Reply:
I added `reg_is_simd()` in order for AArch64 to conform to the API.

Reply:
I see. Would it be possible to add a test for it to https://github.com/DynamoRIO/dynamorio/blob/master/suite/tests/api/opnd-a64.c ?